extensions/amp-facebook-like/amp-facebook-like.md

```diff
@@ -0,0 +1,97 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-facebook-like"></a> `amp-facebook-like`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Embeds the Facebook like button plugin.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Stable</td>
```

Comment:
Update to "In Development" and change to "Stable" 3 weeks after PR is merged (when validator changes are in production)